client/dashboard/sites/overview/index.tsx

```diff
@@ -22,9 +24,55 @@ import StorageCard from './storage-card';
 import UptimeCard from './uptime-card';
 import './style.scss';
 
-function SiteOverview() {
+type Breakpoint = Parameters< typeof useViewportMatch >[ 0 ];
+
+const SPACING = {
+	DEFAULT: 6,
+	SMALL: 4,
+};
+
+function getGridLayout( {
+	count,
+	isLargeViewport,
+	isSmallViewport,
+}: {
+	count: number;
+	isLargeViewport: boolean;
+	isSmallViewport: boolean;
```

Comment:
I feel like these should be combined into

```suggestion
	viewport: 'large' | 'medium' | 'small';
```

But I guess that's not how `useViewportMatch()` works 😢

Reply:
Yeah, let's leave it as it is.